tests/foreman/ui/test_user.py

```diff
@@ -1329,3 +1330,437 @@ def test_positive_create_in_ldap_mode(self):
+    @tier1
+    def test_positive_create_ssh_key_super_admin(self):
+        """SSH Key can be added to Super Admin user details page
+
+        :id: 31388483-35f5-4828-82e9-9305a76e712d
+
+        :steps:
+
+                1. Go to Administer -> Users
+                2. Edit Super Admin user details page
+                3. Add SSH Key in SSH Keys tab
+                4. Save the changes of Super Admin user
+
+        :expectedresults: Super Admin should be saved with SSH key
+        """
+
+    @stubbed
+    @tier1
+    @skip_if_bug_open('bugzilla', 1465389)
+    def test_positive_create_multiple_ssh_keys(self):
+        """Multiple SSH Keys can be added while creating a new user
+
+        :id: 6552194f-63ff-4a6e-9784-5b3dc1772fd5
+
+        :steps:
+
+                1. Go to Administer -> Users
+                2. Attempt to create new user with all the details
+                3. Add multiple SSH Keys in SSH Keys tab before saving the user
+                4. Save the new User
+
+        :expectedresults: New user should be added with multiple SSH keys
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_create_multiple_ssh_keys_super_admin(self):
+        """Multiple SSH Keys can be added to Super admin user details page
+
+        :id: 267cea76-0b75-4b37-a04f-dc3659cab409
+
+        :steps:
+
+                1. Go to Administer -> Users
+                2. Edit Super Admin user details page
+                3. Add multiple SSH Keys in SSH Keys tab
+                4. Save the changes of Super Admin user
+
+        :expectedresults: Super Admin should be saved with multiple SSH keys
+        """
+
+    @stubbed
+    @tier1
+    def test_negative_create_ssh_key(self):
+        """Invalid ssh key can not be added in User details page
+
+        :id: a815cd8b-142e-4743-b95a-c922def193f6
+
+        :steps:
+
+                1. Go to Administer -> Users
+                2. Attempt to create new user with all the details
+                3. Attempt to add invalid string as SSH Key in SSH Keys tab
+                    before saving the user
+
+        :expectedresults: Invalid SSH key should not be added in user details
+            page
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_notify_create_invalid_ssh_key(self):
+        """Satellite Notifies on invalid ssh key creation in user details page
+
+        :id: ea613925-75a0-421c-b02b-e61ce2fe0d84
+
+        :steps:
+
+                1. Go to Administer -> Users
+                2. Attempt to create new user with all the details
+                3. Attempt to add invalid string as SSH Key in SSH Keys tab
+                    before saving the user. e.g blabla
+
+        :expectedresults: Satellite should show 'Fingerprint could not be
+            generated' error notification
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_notify_create_invalid_length_ssh_key(self):
+        """Satellite Notifies on invalid length ssh key creation in user
```

Comment:
"""SSH key with too long length can not be added to user and corresponding error notification should be displayed